imod/mf6/simulation.py

```diff
@@ -989,15 +990,17 @@ def _add_modelsplit_exchanges(self, exchanges_list: List[GWFGWF]) -> None:
     def _set_exchange_options(self):
         # collect some options that we will auto-set
         for exchange in self["split_exchanges"]:
-            model_name_1 = exchange.dataset["model_name_1"].values[()]
-            model_1 = self[model_name_1]
-            exchange.set_options(
-                save_flows=model_1["oc"].is_budget_output,
-                dewatered=model_1["npf"].is_dewatered,
-                variablecv=model_1["npf"].is_variable_vertical_conductance,
-                xt3d=model_1["npf"].get_xt3d_option(),
-                newton=model_1.is_use_newton(),
-            )
+            if isinstance(exchange, GWFGWF):
+                model_name_1 = exchange.dataset["model_name_1"].values[()]
+                model_1 = self[model_name_1]
+            if isinstance(model_1, GroundwaterFlowModel):
+                exchange.set_options(
+                    save_flows=model_1["oc"].is_budget_output,
+                    dewatered=model_1["npf"].is_dewatered,
+                    variablecv=model_1["npf"].is_variable_vertical_conductance,
+                    xt3d=model_1["npf"].get_xt3d_option(),
+                    newton=model_1.is_use_newton(),
+                )
```

Comment:
This works only if ``model_1`` is defined, which is not the case if the exchange is GWFGWT or GWTGWT.

Reply:
True, but gwfgwt exchanges are created when writing the simulation, while gwfgwf's are created when splitting. gwtgwt will be implemented in the next PR.
This function is called at the end of splitting.
It may be a good idea to also migrate the creation of gwfgwt exchanges to the split method, but that's beyond the scope of this PR